# Retain upstream defaults and expose the desktop-file and icon install directories.
# TODO: submit upstream; generated with Python difflib against aerc 0.22.0.
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -14,6 +14,8 @@
 SHAREDIR ?= $(PREFIX)/share/aerc
 LIBEXECDIR ?= $(PREFIX)/libexec/aerc
 MANDIR ?= $(PREFIX)/share/man
+DESKTOPDIR ?= $(PREFIX)/share/applications
+ICONDIR ?= $(PREFIX)/share/icons/hicolor/scalable/apps
 GO ?= go
 INSTALL ?= install
 CP ?= cp
@@ -211,8 +213,8 @@
 	$(eval $(call install_file,$s,644,$(DESTDIR)$(SHAREDIR)/stylesets)))
 $(foreach t,$(wildcard templates/*),\
 	$(eval $(call install_file,$t,644,$(DESTDIR)$(SHAREDIR)/templates)))
-$(eval $(call install_file,contrib/aerc.desktop,644,$(DESTDIR)$(PREFIX)/share/applications))
-$(eval $(call install_file,contrib/aerc.svg,644,$(DESTDIR)$(PREFIX)/share/icons/hicolor/scalable/apps))
+$(eval $(call install_file,contrib/aerc.desktop,644,$(DESTDIR)$(DESKTOPDIR)))
+$(eval $(call install_file,contrib/aerc.svg,644,$(DESTDIR)$(ICONDIR)))
 $(eval $(call install_file,aerc,755,$(DESTDIR)$(BINDIR)))
 $(eval $(call install_file,contrib/carddav-query,755,$(DESTDIR)$(BINDIR)))
 
